SPECS/pytorch/pytorch.spec

```
@@ -0,0 +1,113 @@
+%global _empty_manifest_terminate_build 0
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+Name:           pytorch
+Version:        1.13.1
+Release:        1%{?dist}
+License:        BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://pytorch.org/
+Source0:        https://github.com/pytorch/pytorch/releases/download/v%{version}/pytorch-v%{version}.tar.gz#/pytorch-%{version}.tar.gz
```

For the sake of following what we try to do in most specs:

```suggestion
Source0:        https://github.com/pytorch/pytorch/releases/download/v%{version}/%{name}-v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```